publisher/src/components/DataUpload/UploadedFiles.tsx

```diff
@@ -155,12 +155,18 @@ export const UploadedFileRow: React.FC<{
         <UploadedFilesCell capitalize>
           <UploadedContainer>
             {/* TODO(#334) Hook up admin badges rendering to team member roles API */}
-            <TeamMemberNameWithBadge
-              name={uploadedByName}
-              badgeId={id?.toString()}
-              role={uploadedByRole}
-            />
-            <DateUploaded>{`/ ${dateUploaded}`}</DateUploaded>
+            {uploadedByName && (
+              <TeamMemberNameWithBadge
+                name={uploadedByName}
+                badgeId={id?.toString()}
+                role={uploadedByRole}
+              />
+            )}
+            {uploadedByName ? (
+              <DateUploaded>{`/ ${dateUploaded}`}</DateUploaded>
+            ) : (
+              <>{dateUploaded}</>
+            )}
```

Comment:
What if we simplified this to something like:

```suggestion
            {uploadedByName ? (
            <>
              <TeamMemberNameWithBadge
                name={uploadedByName}
                badgeId={id?.toString()}
                role={uploadedByRole}
              />
              <DateUploaded>{`/ ${dateUploaded}`}</DateUploaded>
              </>
            ) : (
              <>{dateUploaded}</>
            )}
```